server/command_hooks.go

```diff
@@ -25,6 +26,18 @@ func (p *Plugin) registerCommand(teamId string) error {
 		return errors.Wrap(err, "failed to register command")
 	}
 
+	if err := p.API.RegisterCommand(&model.Command{
+		TeamId:           teamId,
+		Trigger:          CommandTriggerCrash,
+		AutoComplete:     true,
+		AutoCompleteHint: "",
+		AutoCompleteDesc: "Crashes plugin.",
+		DisplayName:      "Crashes plugin",
```

Comment:
`DisplayName` and `Description` are not needed. They are only used for "normal" integrations.

Reply:
Removed.